editor.planx.uk/src/@planx/components/NextSteps/model.ts

```diff
@@ -3,15 +3,21 @@ import { MoreInformation, parseMoreInformation } from "../shared";
 export interface NextSteps extends MoreInformation {
   title: string;
   description: string;
-  fn: string;
+  steps: Array<Step>;
 }
 
-export const parseContent = (
-  data: Record<string, any> | undefined,
+export interface Step {
+  title: string;
+  description: string;
+  url: string;
+}
+
+export const parseNextSteps = (
+  data: Record<string, any> | undefined
 ): NextSteps => ({
+  steps: /* remove once migrated */ data?.taskList?.steps || data?.steps || [],
```

Comment:
it looks to me like this could just be: `steps: data?.steps || [],` ? 

i don't see any other references to `taskList` & contrary to the code comment it looks like we definitely still want the `steps` property defined overall because it's required on the `NextSteps` interface defined above.

Reply:
Updated, thanks for the explanation here.